source/index.txt

```diff
@@ -1,80 +1,257 @@
+:template: product-landing
+:hidefeedback: header
+:noprevnext:
+
 .. program:: mongosh
 
 .. binary:: mongosh
 
 .. _mdb-shell-overview:
 
-===========================
-MongoDB Shell (``mongosh``)
-===========================
+======================================
+Welcome to MongoDB Shell (``mongosh``)
+======================================
+
+.. introduction::
+
+   The |mdb-shell|, ``mongosh``, is a JavaScript and Node.js |repl| 
+   environment for interacting with MongoDB deployments in 
+   `Atlas <https://www.mongodb.com/docs/atlas/>`__, locally,
+   or on another remote host. Use the |mdb-shell| to test queries and 
+   interact with the data in your MongoDB database.
+
+   .. button:: Download mongosh
+      :uri: https://www.mongodb.com/try/download/shell?jmp=docs
+
+.. image:: /images/mongosh-plp/Technical_SOFTWARE_Terminal(3)_Spot.png
+   :alt: Abstract illustration showing a terminal interacting with different systems 
+
+.. kicker:: What You Can Do
+
+Access MongoDB From Your Shell
+------------------------------
+
+.. tabs::
+
+   .. tab:: Connect to MongoDB
+      :tabid: connect
+
+      .. procedure::
+
+         .. step:: Find Your Connection String
+
+            Find your connection string. The connection string varies 
+            depending on the type of deployment you're connecting to.
+
+            Learn how to find your connection string on
+            `Atlas <https://www.mongodb.com/docs/manual/reference/connection-string/#find-your-connection-string>`__.
+
+            Or connect to an 
+            `on-premise <https://www.mongodb.com/docs/manual/reference/connection-string/#find-your-self-hosted-deployment-s-connection-string>`__
+            deployment.
+
+         .. step:: Connect to MongoDB
+
+            Connect to a MongoDB deployment in Atlas using the Atlas 
+            connection string:
```

Comment:
Suggestion: tweak language so that it's relevant to both Atlas and on prem, but keep the Atlas example
```suggestion
            Connect to a MongoDB deployment using the
            connection string. 
            
            The following connection string connects to 
            an Atlas deployment:
```